anycloud/node/src/datastore.ts

```diff
@@ -48,7 +48,7 @@ class DataStore {
       try {
         dsValue = await this.request('GET', `${this.ctrlPortUrl}get/${dsKey}`);
       } catch (e) {
-        return Error(e);
+        return new Error(e);
```

Comment:
Isn't `e` already an Error object? Why the rewrapping?

Reply:
you are completely right, my bad.